drivers/axi_core/jesd204/axi_adxcvr.c

```diff
@@ -380,7 +380,7 @@ int32_t adxcvr_status_error(struct adxcvr *xcvr)
  */
 static int adxcvr_reset(struct adxcvr *xcvr)
 {
-	int ret, retry = 1;
+	int ret = 0, retry = 1;
```

Comment:
Wouldn't it be better to initialize with error value? It would be like an additional check that the routine is executed correctly.